src/eth/eth-wallet.js

```diff
@@ -191,16 +190,14 @@ class EthWallet {
   }
 
   setLatestBlock (blockNumber) {
-    this._latestBlock = blockNumber;
+    this._latestBlock = Math.max(this._latestBlock, blockNumber);
```

Comment:
The server now guarantees never sending older blocks than have already been sent, so this is not necessary.

Reply:
Nice, will remove.